packages/components/src/components/tag/tag.tsx

```diff
@@ -32,15 +32,15 @@ export class Tag implements Base {
   /** decorator Jss stylesheet */
   @CssInJs('Tag', styles) stylesheet: StyleSheet;
 
-  @Event() close: EventEmitter<any>;
+  @Event() scaleClose: EventEmitter<any>;
```

Comment:
Can we add types? See prev comment...

Reply:
Please add description for docs, see pre comment...

Reply:
OK!